app/services/ReviewService.scala

```diff
@@ -64,31 +64,61 @@ class ReviewService @Inject() (publishedService: PublishedService, repository: A
   def changeStatus(id: String, currentStatus: String, info: ApprovalProcessStatusChange): Future[RequestOutcome[Unit]] = {
 
     def getContentToUpdate: Future[Option[ApprovalProcess]] = repository.getById(id) map {
-        case Right(process) => if (process.meta.status == currentStatus) Some(process) else None
-        case _ => None
-      }
+      case Right(process) =>
+        if (process.meta.status == currentStatus) {
+          Some (process)
+        }
+        else {
+          logger.warn(s"Invalid Process Status Change requested for process $id: " +
+            s"Expected Status: '$currentStatus' Status Found: '${process.meta.status}' Desired Status: ${info.status} ")
+          None
+        }
+      case _ =>
```

Comment:
Could be a db err from the getById() call

Reply:
restructured to handle errors